sycl/plugins/opencl/pi_opencl.cpp

```diff
@@ -716,8 +716,16 @@ pi_result piEventCreate(pi_context context, pi_event *ret_event) {
 }
 
 pi_result piextEventCreateWithNativeHandle(pi_native_handle nativeHandle,
+                                           pi_context context,
+                                           bool ownNativeHandle,
                                            pi_event *piEvent) {
+  (void)ownNativeHandle;
```

Comment:
Probably, there should be `(void)smth` for other unused fields as `asserts` may become no-ops.

Reply:
OK